dvc/remote/gdrive.py

```diff
@@ -104,13 +104,16 @@ def init_drive(self):
                 ),
             )
         )
+        self.corpora = "allDrives"
```

Comment:
do we need this default value? if not it would be safer to keep them None, to make sure that we don't hit `allDrive` after some changes in the future.

Reply:
yep, we need `allDrives` initially, since we don't know initially where the root directory is located - in shared or user's drive "collection".

Reply:
do we hit any APIs that require `corpora` though before we able to figure out this?

Reply:
well, it looks like it might occur. I will look into possibility to put the assert into PyDrive2 to catch this.

Reply:
not into PyDrive, probably 🤔  ... I meant that I still don't quite understand why do we need this default ... it's very close to the beginning where we get the actual value for it - default or drive, right? so why do we need this default? is it needed for the period we are still not sure is a shared drive or not? do we hit any APIs that require this value?

Reply:
@shcheklein sorry, I meant, that, yes, currently ListFile API is called. But it is possible to avoid this, will update the code.

Reply:
gotcha, yep, let's avoid this if possible and remove the default value